project/ScalaJUnitMixinPlugin.scala

```diff
@@ -0,0 +1,27 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js sbt plugin        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+import sbt.Keys._
+import sbt._
+
+object ScalaJUnitMixinPlugin extends AutoPlugin {
+
+  override def projectSettings: Seq[Setting[_]] = Seq(
+    libraryDependencies in Test ++=
```

Comment:
`libraryDependencies in Test` is never used. See:
```
> set libraryDependencies in (compiler, Test) += "org.blop" % "sfoih" % "1.0"
[info] Defining compiler/test:libraryDependencies
[info] The new value will be used by no settings or tasks.
[info] Reapplying settings...
```
We need a configuration like the one in https://github.com/scala-js/scala-2.12-junit-mixin-plugin/pull/13

Btw, I still think there is no good reason for this config to be in a separate file. It can just be a `val` within `Build`. Note in particular that, currently, the `AutoPlugin` nature of `ScalaJUnitMixinPlugin` is not even used. So this is really just an `object` with two `def`s, which can and should be in `Build`.